### PR TITLE
update nodejs example

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: nodejs-example/oid4vc-example.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  const credentialProvider = createCredentialProvider({
+    wallet,
+  });
+
+  console.log('Fetching the credential from the OID4VC issuer');
+
+  // OID4VC Url
+  // This URL can be rendered as a QR Code
+  const credentialUrl = 'openid-credential-offer://?credential_offer=%7B%22credential_issuer%22%3A%22https%3A%2F%2Fapi-staging.dock.io%2Fopenid%2Fissuers%2F2baff124-6681-428b-b5a1-449f211d9624%22%2C%22credentials%22%3A%5B%22ldp_vc%3AMyCredential%22%5D%2C%22grants%22%3A%7B%22urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Apre-authorized_code%22%3A%7B%22pre-authorized_code%22%3A%22bN3fNhSP0TnVoGgYzn0lOsVq-hQCmelt3AvDa-TA3n0%22%2C%22user_pin_required%22%3Afalse%7D%7D%7D';
```
Comment (mike-parkhill) [inline nodejs-example/oid4vc-example.js]: should we describe where this comes from more? Or link to the docs about it or something?

Comment (maycon-mello) [reply nodejs-example/oid4vc-example.js]: good point, yes let me do that

File: nodejs-example/generate-oid4vc-offer.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    const response = await apiClient.post("/openid/issuers", {
+      claimMap: { name: "name" },
+      credentialOptions: {
+        credential: {
+          type: ["MyCredential"],
+          subject: { id: "test", test: "test" },
+          issuanceDate: "2024-08-09T14:15:22Z",
+          expirationDate: "2099-08-24T14:15:22Z",
+          schema: "https://schema.dock.io/Test-V1-1701450859791.json",
+          issuer: "did:dock:5CkQ14QjBuuidYJPC7uRMhN1Hrtv3kTLuUx8dGsnNgm4RtT5",
```
Comment (mike-parkhill) [inline nodejs-example/generate-oid4vc-offer.js]: they'll need to specify an issuer DID that they control, right?

File: nodejs-example/generate-oid4vc-offer.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    return response.data;
+  } catch (error) {
+    console.error("Failed to create OpenID issuer:", error);
+  }
+}
+
+async function generateOID4VCOffer() {
+  try {
+    const response = await apiClient.post("/openid/credential-offers", {
+      id: "2baff124-6681-428b-b5a1-449f211d9624",
```
Comment (mike-parkhill) [inline nodejs-example/generate-oid4vc-offer.js]: is it okay for this id to be hard-coded?

Comment (maycon-mello) [reply nodejs-example/generate-oid4vc-offer.js]: should not be

File: nodejs-example/generate-oid4vc-offer.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    });
+    console.log(`OID4VC offer ${response.data.id} was created.`);
+    console.log(`Copy your OID4VC URL: ${response.data.url}`);
```
Comment (mike-parkhill) [inline nodejs-example/generate-oid4vc-offer.js:48]: could we export it to the ENV automatically?

Comment (maycon-mello) [reply nodejs-example/generate-oid4vc-offer.js:48]: I think so

Comment (maycon-mello) [reply nodejs-example/generate-oid4vc-offer.js:48]: I would need to configure env files and dot-env configs. I wonder if that will not make the example more complex and harder to understand

File: nodejs-example/generate-oid4vc-offer.js (hunk 3 above)
Comment (maycon-mello) [reply nodejs-example/generate-oid4vc-offer.js]: fixed

File: nodejs-example/generate-oid4vc-offer.js (hunk 2 above)
Comment (maycon-mello) [reply nodejs-example/generate-oid4vc-offer.js]: fixed

File: nodejs-example/generate-oid4vc-offer.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
+const axios = require("axios").default;
+
+const API_KEY = process.env.CERTS_API_KEY;
+const API_URL = process.env.CERTS_API_URL;
+const ISSUER_DID = process.env.ISSUER_DID;
```
Comment (mike-parkhill) [inline nodejs-example/generate-oid4vc-offer.js:5]: maybe we should add a comment telling them where to get this from Certs or the API
* https://certs.dock.io/dids
* $API_URL/dids

**Review by mike-parkhill — APPROVED** [review]

File: nodejs-example/generate-oid4vc-offer.js (hunk 5 above)
Comment (mike-parkhill) [reply nodejs-example/generate-oid4vc-offer.js:5]: I added text to the readme for this

**Pull request merged**